README.md

```diff
@@ -1,2 +1,55 @@
 # ops-utils
-This is a very small docker image with some useful operation utils (like rclone, mysql-client, postgresql-client, wait-for-it)
+
+This is a very small docker image with some useful operation utils (like rclone, mysql-client, postgresql-client, wait-for-it).
+
+## MySQL
+
+### Import
+
+```bash
+docker run --rm ops-utils:latest mysql import
```

Comment:
I would be more clear to be call it “Import-from-bucket”

Reply:
Done